subsys/mgmt/mcumgr/transport/src/smp_udp.c

```diff
@@ -133,14 +164,78 @@ static int smp_udp_ud_copy(struct net_buf *dst, const struct net_buf *src)
 	return MGMT_ERR_EOK;
 }
 
+static int create_socket(enum proto_type proto)
```

Comment:
It would be good to separate the return code (ie. error) from the socket value.
```suggestion
static int create_socket(enum proto_type proto, int* socket)
```

Reply:
Fixed